cmd/keycloakb/keycloak_bridge.go

```diff
@@ -312,6 +314,13 @@ func main() {
 		}
 	}
 
+	// Security - AES encryption mechanism for users PII
```

Comment:
Would be good to have a check which ensure the encryption key is not empty

Reply:
Isn't there an automatic test of encrypt/decrypt using the key ?

Reply:
We are performing an encryption/decryption cycle using the key in NewAesGcmEncrypterFromBase64(...)
That's why there is a returned error. I suppose that if the encryption key is empty, the encryption/decryption cycle should fail.

Reply:
No, actually if the key is "" then the decoding succeeds (I just checked). I am adding a modification on the common-service to ensure that the correct length of the key is provided.

Reply:
If we do it at startup it would avoid to fail while performing the operation, that's why it could be interesting to have such check. As you want not an hard requirement

Reply:
The check is already done in NewAesGcmEncrypterFromBase64(...)